pkg/tokens/builtin/zone/token.go

```diff
@@ -0,0 +1,37 @@
+package zone
+
+import (
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/pkg/errors"
+
+	core_model "github.com/kumahq/kuma/pkg/core/resources/model"
+	core_tokens "github.com/kumahq/kuma/pkg/core/tokens"
+)
+
+const SigningKeyPrefix = "zone-token-signing-key"
+const SigningKeyPublicSuffix = ".pub"
```

Comment:
I'd change `.` to `-`. Dot has a special meaning for how we look at kubernetes resources. Also, I think the suffix will be a bit annoying with the rest of token machinery, look at `SigningKeyResourceKey(` for example.
I have a feeling that `zone-token-public-key` will be a more convenient choice.

Reply:
addressed